src/ButterCMS/ButterCMS.php

```diff
@@ -126,6 +134,22 @@ public function fetchTags($params = [])
     }
 
     ///////////////
+    // Pages
+    ///////////////
+
+    public function fetchPage($type, $slug, array $params = [])
+    {
+        $rawPage = $this->request('pages/' . $type . '/' . $slug . '/', $params);
```

Comment:
Just confirming, this results in `/pages/<type>/<slug>/?foo1=bar1&foo2=bar2` , correct?

Reply:
Yeah, that's what `'query' => $params` above on the Guzzle client does. I assume it basically uses [http_build_query()](http://php.net/manual/en/function.http-build-query.php) behind the scenes.